packages/api/src/derives/fees/estimateFee.ts

```diff
@@ -36,7 +36,7 @@ import { catchError, first, map, switchMap } from 'rxjs/operators';
 //    const maxPayment = 'xxx' // Max amount user is ok to pay in 'fee asset'
 //    const feeFromQuery = await api.derive.fees.estimateFee({extrinsic, userFeeAssetId, maxPayment}) // this will only be successful if their is enough liquidity of users asset in exchange pool
 
-export function estimateFee(instanceId: string, api: ApiInterfaceRx) {
+export function estimateFee(instanceId: string, api: ApiInterfaceRx): any {
```

Comment:
can we try some ways to specify type for this... 
Observable<?> or Observable<any>

Reply:
yup seems possible just need to change to a function signature that returns Observable